.github/workflows/rust.yml

```
@@ -3,22 +3,47 @@ on: [push, pull_request]
 name: Continuous integration
 
 jobs:
-  bench_nightly:
-    name: Nightly - ASan + Bench
+  Tests:
+    name: Feature matrix tests
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        rust:
-          - nightly
+        include:
+          - rust: stable
+          - rust: beta
+          - rust: nightly
+          - rust: 1.29.0
```

This is equivalent to:

```suggestion
        rust: [stable, beta, nightly, 1.29.0]
```

---

See https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idstrategymatrix.

---

Used as suggested, cheers.